src/arch/win/unix2nt_cc

```diff
@@ -206,11 +211,12 @@ do
 		CL_OPTS+=' -DNDEBUG -Ox'
 		ML_OPTS+=' -DNDEBUG'
 		;;
-	-g)
+	-g|-g1|-g2|-g3)
```

Comment:
maybe write as `-g|-g?|-ggdb`. XLC supports up to -g9.

Reply:
Done